src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -116,10 +149,11 @@
      (swap! state dissoc :validation-error :server-error)
      (let [validation-errors (atom [])
            field-data (->> props :library-schema :properties
-                           (keep (fn [[property-key {:keys [hidden enum] :as property}]]
+                           (keep (fn [[property-key {:keys [hidden enum typeahead] :as property}]]
                                    (let [pk-str (name property-key)
-                                         value (cond hidden (resolve-hidden property-key (:workspace props))
+                                         value (cond hidden (resolve-hidden property-key (:workspace props) (:ontologyDOID @state))
                                                      enum (resolve-enum (common/get-text refs pk-str))
+                                                     (= typeahead "ontology") (:ontologyText @state)
```

Comment:
add a TODO comment: when we support multiple ontology-driven properties, can't rely on hardcoded values in state.